woocommerce/woocommerce-gutenberg-products-block
# Move checkout terms & conditions/privacy policy tests

## What

Moves checkout terms & conditions/privacy policy tests to Playwright.

Fixes #11898

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Run CI and ensure tests pass.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
